fibonacci-without-recursive.js

```diff
@@ -0,0 +1,23 @@
+// Napisz funkcję fibonacci(n), która, nie używając rekurencji, zwróci n-ty wyraz Ciągu Fibonacciego.
+
+function fibonacci(n) {
+    if (n < 2) {
+        return n;
+    }
+
+    let a = 0,
+        b = 1,
+        c = 1;
```

Comment:
Yes, because the value of variable has to be always the same in this task. Of cours this is my opinion